NOTES about this Chainguard created patch:

This patch bumps one of the gradle dependencies. This is one of two steps
required. After bumping any gradle dependencies, you need to re-generate the
gradle-verification-metadata, and also create a patch for that:

```bash
./gradlew help --no-daemon --write-verification-metadata pgp,sha256 --export-keys
```

Also see the upstream docs:
- https://github.com/elastic/elasticsearch/blob/main/BUILDING.md

---

diff --git a/x-pack/plugin/security/build.gradle b/x-pack/plugin/security/build.gradle
index 07308d5d29a..669b6215ee5 100644
--- a/x-pack/plugin/security/build.gradle
+++ b/x-pack/plugin/security/build.gradle
@@ -80,7 +80,7 @@ dependencies {
 
   // Dependencies for oidc
   api "com.nimbusds:oauth2-oidc-sdk:9.37"
-  api "com.nimbusds:nimbus-jose-jwt:9.23"
+  api "com.nimbusds:nimbus-jose-jwt:9.37.2"
   api "com.nimbusds:lang-tag:1.4.4"
   api "com.sun.mail:jakarta.mail:1.6.3"
   api "net.jcip:jcip-annotations:1.0"
@@ -103,7 +103,7 @@ dependencies {
   testImplementation('org.apache.kerby:kerb-crypto:1.1.1')
   testImplementation('org.apache.kerby:kerb-util:1.1.1')
   testImplementation('org.apache.kerby:token-provider:1.1.1')
-  testImplementation('com.nimbusds:nimbus-jose-jwt:9.23')
+  testImplementation('com.nimbusds:nimbus-jose-jwt:9.37.2')
   testImplementation('net.jcip:jcip-annotations:1.0')
   testImplementation('org.apache.kerby:kerb-admin:1.1.1')
   testImplementation('org.apache.kerby:kerb-server:1.1.1')